internal/authctx/helper.go

```diff
@@ -76,14 +82,45 @@ func ProviderAuthSchema() map[string]*schema.Schema {
 	}
 }
 
+var selfManagedAuthSchema = &schema.Schema{
+	Type:     schema.TypeList,
+	Optional: true,
```

Comment:
Kindly add descriptions to all the components in this schema.

Reply:
Done, I have added the descriptions.